README.md

```diff
@@ -1675,6 +1675,16 @@ Table of Contents
   * [Rive](https://rive.app/community/) - Community assets as well as create your own game assets using its free plan.
   * [Poly Pizza](https://poly.pizza/) - Free low poly 3D assets
   * [3Dassets.one](https://www.3dassets.one/) - Over 8,000 free/paid 3D models, and PBR materials for making textures.
+  * [Unity Asset Store](https://assetstore.unity.com/) - Thousands of free or affordable game assets (textures/models/tools/etc)
+  * [Godot Engine](https://godotengine.org/) - Cross-platform, free, and open-source game engine. For making 2D and 3D games.
+  * [Love2D](https://love2d.org/) - LÖVE is an open-source cross-platform engine for developing 2D video games.
+  * [ambientCG](https://ambientcg.com/) - Public domain materials and resources, for physically-based rendering.
+  * [Blender](https://www.blender.org/) -  Free and open-source 3D graphics toolset, used for creating 3D models, animations, and VR.
+  * [Inkscape](https://www.inkscape.org/) - Free and open-source vector graphics editor used to create vector images, primarily in SVG format.
+  * [Audacity](https://www.audacityteam.org/) - Free and open-source audio editor and recording software. (Windows/Mac/Linux)
+  * [LMMS](https://lmms.io/lsp/) - Free, open-source, multiplatform digital audio workstation.
```

Comment:
This is a list for SaaS so while I get you might consider these on par with development IDEs, given the scope of the list stated top of the readme I am inclined to say non of these are a good fit - apart from the asset store